test/vcr_python_wrapper.py

```diff
@@ -41,6 +41,21 @@ def domain_query_matcher(r1, r2):
     return r1.query == r2.query
 
 
+def body_json_host_matcher(r1, r2):
+    if r1.headers.get('content-type') == 'application/json' and r2.headers.get('content-type') == 'application/json':
+        if r1.path == '/api/v2/hostgroups' and r2.path == '/api/v2/hostgroups':
```

Comment:
This can be written as `r1.path == r2.path == '/api/v2/hostgroups'` which is shorter and IMHO more readable.